docs/rules/symbol-description.md

```diff
@@ -0,0 +1,94 @@
+# Require/Disallow Symbol Description (symbol-description)
+
+The `Symbol` constructor may have optional description:
```

Comment:
@jrencz I would recommend fixing those docs in a separate issue/PR (if you weren't already thinking of doing that)